dali/kernels/kernel.h

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2018, NVIDIA CORPORATION. All rights reserved.
+// Copyright (c) 2018, 2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
It is just a question, but is there a particular reason to use `2018, 2021` instead of `2018-2021`? For me it is the some, but maybe there is some meaning behind this?